core-java-modules/core-java-jndi/src/main/java/com/baeldung/jndi/ldap/connection/tool/LdapConnectionTool.java

```diff
@@ -0,0 +1,116 @@
+package com.baeldung.jndi.ldap.connection.tool;
+
+import java.util.Hashtable;
+
+import javax.naming.Context;
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.Attribute;
+import javax.naming.directory.Attributes;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.InitialDirContext;
+
+public class LdapConnectionTool {
+    private static boolean DEBUG_MODE = Boolean.valueOf(System.getProperty("debug.mode", "false"));
```

Comment:
Seeing the capitalization I think you want to set the debug mode as a constant, if that´s the case you have to add the final modifier, otherwise, you have to correct the capitalization.